src/index.d.ts

```diff
@@ -994,10 +994,12 @@ declare namespace admin.remoteConfig {
     /**
      * Gets the current active version of the {@link admin.remoteConfig.RemoteConfigTemplate
      * `RemoteConfigTemplate`} of the project.
-     *
+     * 
+     * @param versionNumber Version number of the Remote Config template to look up.
+     *    If not specified, the latest Remote Config template will be returned. Optional.
```

Comment:
Always prefer "is" to "will be" if it works.  It does here!  :)

Reply:
Good catch! Thank you. Updated!